src/game/common/gamemain.cpp

```diff
@@ -31,6 +33,8 @@ void Game_Main(int argc, char *argv[])
 {
     g_theGameEngine = Create_Game_Engine();
     g_theGameEngine->Init(argc, argv);
+    // this should happen in GameEngine::Init but we don't have that yet
```

Comment:
Mark it as TODO to make it more discoverable.

Reply:
Fixed.